node/network/protocol/src/lib.rs

```diff
@@ -357,6 +357,14 @@ pub mod v1 {
 			}
 		}
 
+		/// Get the signature from the statement.
+		pub fn get_signature(&self) -> ValidatorSignature {
```

Comment:
nit: I'd commonly expect this to return a `&ValidatorSignature` and allow the user to clone as needed, here it only has a single use case, so up to you